src/components/MapView/Layers/raster-utils.ts

```diff
@@ -120,21 +121,57 @@ export function getWCSUrl(
   return formatUrl(baseUrl, params);
 }
 
+export function getWCS2Url(
```

Comment:
Let's use the same formatting between v1 and v2 name. To make it clearer, how about:

- getWCSv1Url
- getWCSv2Url

?

Reply:
Included in 33a1a5de12ba3f9f592e755c5a5b5eabfc53d947